ts/nni_manager/test/extensions/experimentManager.test.ts

```diff
@@ -1,60 +1,56 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
 
-'use strict';
-
 import { assert, expect } from 'chai';
-import * as fs from 'fs';
+import fs from 'fs';
 import { Container, Scope } from 'typescript-ioc';
+import os from 'os';
+import path from 'path';
 
 import * as component from '../../common/component';
 import { cleanupUnitTest, prepareUnitTest } from '../../common/utils';
-import { ExperimentManager } from '../../common/experimentManager';
-import { NNIExperimentsManager } from 'extensions/experiments_manager';
-
-
-describe('Unit test for experiment manager', function () {
-    let experimentManager: NNIExperimentsManager;
-    const mockedInfo = {
-        "test": {
-            "port": 8080,
-            "startTime": 1605246730756,
-            "endTime": "N/A",
-            "status": "INITIALIZED",
-            "platform": "local",
-            "experimentName": "testExp",
-            "tag": [], "pid": 11111,
-            "webuiUrl": [],
-            "logDir": null
-        }
+import { ExperimentsManager } from 'extensions/experiments_manager';
+import globals from 'common/globals/unittest';
+
+let tempDir: string | null = null;
+let experimentManager: ExperimentsManager;
+const mockedInfo = {
+    "test": {
+        "port": 8080,
+        "startTime": 1605246730756,
+        "endTime": "N/A",
+        "status": "INITIALIZED",
+        "platform": "local",
+        "experimentName": "testExp",
+        "tag": [], "pid": 11111,
+        "webuiUrl": [],
+        "logDir": null
     }
+}
 
+describe('Unit test for experiment manager', function () {
     before(() => {
         prepareUnitTest();
-        fs.writeFileSync('.experiment.test', JSON.stringify(mockedInfo));
-        Container.bind(ExperimentManager).to(NNIExperimentsManager).scope(Scope.Singleton);
-        experimentManager = component.get(NNIExperimentsManager);
-        experimentManager.setExperimentPath('.experiment.test');
+        tempDir = fs.mkdtempSync(path.join(os.tmpdir(), 'nni-ut-'));
```

Comment:
`nni-ut-`?

Reply:
This is directory fix, `tempDir` will be like `/tmp/nni-ut-ri13j`